### ISSUE TITLE
Zenmap doesn't respect the new -v0 option for Nmap

### ISSUE THREAD

**Issue by dmiller-nmap** [nmap/nmap#601]
In #265, we added `-v0` to mean "close STDOUT and produce no output." But providing this via Zenmap results in the `-v` option being removed entirely. Part of the problem is that parts of `zenmapCore/NmapOptions.py` are under the assumption that `-v` cannot take an integer option. Not all of it, though, since `-v2` works, for instance.

**Comment by Varunram** [comment]
Does it accept integer arguments? We are actually converting -v2 given to -v -v right?

**Comment by dmiller-nmap** [comment]
@Varunram Yes, Nmap will accept `-v2`, `-v3`, and so forth. Zenmap may be converting to separate `-v` options, which is probably part of the problem: `-v0` would be converted to 0 `-v` instances, when it should instead be passed through directly.

**Comment by fyodor** [comment]
This was fixed a while back with https://github.com/nmap/nmap/pull/626
